test/GameControllerTest.java

```diff
@@ -1,42 +1,76 @@
+import controllers.GameController;
 import controllers.routes;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import play.Application;
+import play.api.test.FakeRequest;
 import play.inject.guice.GuiceApplicationBuilder;
+import play.mvc.Http;
+import play.mvc.Http.RequestBuilder;
 import play.mvc.Result;
 import play.test.WithApplication;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 import static play.mvc.Http.Status.OK;
+import static play.mvc.Http.Status.SEE_OTHER;
```

Comment:
Are the following imports required?
import static play.mvc.Http.Status.SEE_OTHER;
import play.mvc.Http.RequestBuilder;